src/Enforce/runnables/rules/size/length_equals/spec.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+import { expect } from 'chai';
+
+import lengthEquals from './index';
+
+describe('Tests lengthEquals rule', () => {
+
+    describe('First argument is array or string', () => {
+
+        describe('When first argument is equal to a given value', () => {
+
+            it('Should return true for an array equal to 5', () => {
+                expect(lengthEquals([1, 2, 3, 4, 5], 5)).to.equal(true);
```

Comment:
For input variaty across tests, you can generate an array of random length using `new Array()`, and pass in a random integer as the argument. For example:

```js
const length = faker.random.number();

expect(lengthEquals(new Array(length), length)).to.equal(true);
```